adapters/has_component_adapter.py

```diff
@@ -0,0 +1,93 @@
+from adapters.compound_adapter import CompoundAdapter
+import rdflib
+from owlready2 import *
+from tqdm import tqdm
+from adapters import Adapter
+from biocypher._logger import logger
+from adapters import *
+
+
+class HasComponentEdge(CompoundAdapter):
+
+    def __init__(self, filepath=None, label="has_component", dry_run=False):
+        super().__init__(filepath=filepath, label=label, dry_run=False)
+        self.type = "edge"
+        self.compound_adapter_instance = CompoundAdapter(filepath=filepath, label=label, dry_run=dry_run)
+        print(super().ONTOLOGIES)
```

Comment:
Am afraid you have to refactor most of this codes if they are dependent on the Compound adapter. I have changed that adapter so much. For example, we don't have an `ONTOLOGIES` variable in there, I have changed it to `COMPOUNDS` which makes more sense.

Reply:
> Am afraid you have to refactor most of this codes if they are dependent on the Compound adapter. I have changed that adapter so much. For example, we don't have an `ONTOLOGIES` variable in there, I have changed it to `COMPOUNDS` which makes more sense.

Noted. I have refactored it and made it to work independently.